src/borg/archiver.py

```diff
@@ -4223,16 +4223,41 @@ def define_borg_mount(parser):
 
         .. man NOTES
 
-        The following keys are available for ``--format``:
+        The FORMAT specifier syntax
+        +++++++++++++++++++++++++++
 
+        The ``--format`` option uses python's `format string syntax
+        <https://docs.python.org/3.8/library/string.html#formatstrings>`_.
+
+        Examples:
+        ::
+
+            $ borg list --format '{archive}{NL}' /path/to/repo
+            ArchiveFoo
+            ArchiveBar
+            ...
+
+            # {VAR:NUMBER} - pad to NUMBER columns.
+            # Note: most fields are left-aligned, but some are right-aligned.
```

Comment:
Can this be told more precisely?

Like strings are so, number are so. Or by giving the default format string? Or by pointing at the help output, which display the default format string.